frontend/pendingTranslations.ftl

```diff
@@ -4,7 +4,11 @@
 
 waitlist-heading-2 = Join the { -brand-name-relay-premium } waitlist
 waitlist-submit-label-2 = Join the waitlist
+waitlist-privacy-policy-agree-2 = By clicking “{ waitlist-submit-label-2 }”, you agree to our <a>Privacy Policy</a>.
```

Comment:
Question (non-blocking): Is the anchor tag here incomplete or are you doing some FTL magic?

Reply:
The magic is here: https://github.com/mozilla/fx-private-relay/blob/e7dd5e8014fef1f95e49571824bcc48ced5c1340/frontend/src/pages/vpn-relay/waitlist.page.tsx#L15-L23

Basically, I might as well have called that tag `<privacy-policy-link>`, and then in the `elems` prop set that to:

```tsx
       {
          "privacy-policy-link": (
            <a
              href="https://www.mozilla.org/privacy/firefox-relay/"
              target="_blank"
              rel="noopener noreferrer"
            />
          ),
        }
```

(I have an accepted pull request on `@fluent/react` that makes it easier to do that, but that hasn't been released yet unfortunately...)